### PR TITLE
Update setup.rst

### PR DESCRIPTION
**Pull request by jagonzaar**
I put some code that resolve some problems in windows 10 with xampp, without using composer.

### CODE REVIEW

File: setup.rst [hunk 1]
```diff
@@ -32,6 +32,9 @@ executable that needs to be installed on your system only once:
 
     # Windows systems
     c:\> php -r "readfile('https://symfony.com/installer');" > symfony
+    
+    # Windows 10 and xampp (whit the "PATH" environment variable defined in "c:\xampp\php'
+    c:\> php.exe -r "readfile('https://symfony.com/installer');" > symfony
```
Comment (wouterj) [inline setup.rst]: not needed, when `c:\xampp\php` is in your PATH, `php` will be resolved to `c:\xampp\php\php.exe` by Windows

File: setup.rst [hunk 2]
```diff
@@ -52,6 +55,9 @@ executable that needs to be installed on your system only once:
         # ... then, execute the command as
         c:\> cd projects
         c:\projects\> php symfony
+        # in windows 10 with xampp...
+        c:\> cd projects
+        c:\projects\> php.exe symfony
```
Comment (wouterj) [inline setup.rst]: same applies here (and below)

File: setup.rst [hunk 3]
```diff
@@ -72,6 +81,19 @@ to meet those requirements.
 
     If the installer doesn't work for you or doesn't output anything, make sure
     that the PHP `Phar extension`_ is installed and enabled on your computer.
+    In windows 10 with xampp, if you have un error like
```
Comment (wouterj) [inline setup.rst]: Typo: "un" should be "an"

**Comment by iltar** [comment]
Isn't it possible to simply call `php` instead of `php.exe`? Seems like an issue within xampp and windows 10, not in the docs.

File: setup.rst (hunk 3 above)
Comment (wouterj) [inline setup.rst]: I think this is an error that occurs in all cases on Windows. So let's make a bit more generic:

```rst
.. note::

    If you're on Windows, you might get this error:

        cURL error 60: SSL certificate problem: unable to get local issuer certificate.

    In this case, download this link: http://curl.haxx.se/ca/cacert.pem and move the
    file on your computer. Then edit your `php.ini` file and insert:

    .. code-block:: ini

        curl.cainfo = "C:\path\to\cacert.pem"
```

**Review by wouterj — CHANGES_REQUESTED** [review]

**Comment by jagonzaar** [comment]
Thanks for the idea. I tried with php (when the path is defined) instead of php.exe and that's works...

File: setup.rst (hunk 1 above)
Comment (jagonzaar) [inline setup.rst]: You are right... I used php (not php.exe) and that's works...

File: setup.rst (hunk 2 above)
Comment (jagonzaar) [inline setup.rst]: You are right... I used php (not php.exe) and that's works...
The only problem is that
c:\projects\> php symfony
doesn't create another file. I need use "php symfony" each time that must use symfony.
When I try symfony alone, I have errors:
c:\xampp\htdocs\symfonywebs>symfony
'symfony' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.
the file 'symfony' is the PATH directory and in the directory where I execute that command.
What can be the problem?

File: setup.rst (hunk 3 above)
Comment (jagonzaar) [inline setup.rst]: Sorry... is my "spanglish"!

File: setup.rst (hunk 2 above)
Comment (jagonzaar) [inline setup.rst]: correction: the file symfony is in the PATH....

**Review by javiereguiluz — COMMENTED** [review]
The `setup.rst` article is probably the most critical article for us, because it's usually the entry point for Symfony newcomers. So we must be extra careful when making any changes.

I agree with @wouterj's proposal to add a note about the certificate problems. But I wonder if we need to add the other changes about Windows 10 and XAMPP.

**Comment by javiereguiluz** [comment]
@jagonzaar thanks for contributing these improvements. However, as Wouter explained, most of the changes are not required because `php` will also work on Windows systems. That's why we can close this as "won't fix" ... except for the cURL certificate problem, which is very important/common ... and not only for Windows, so I've opened #7492 to fix it. Thanks!

**Pull request closed without merging**